bin/mock-replay

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+from sentry.runner import configure
+
+configure()
+
+import datetime
+import pathlib
+
+import requests
+from django.conf import settings
+
+from sentry.models import File, Organization, Project
+from sentry.replays.models import ReplayRecordingSegment
+from sentry.replays.testutils import mock_replay
+
+
+def store_replay(replay):
+    response = requests.post(settings.SENTRY_SNUBA + "/tests/replays/insert", json=[replay])
+    assert response.status_code == 200
+
+
+def create_recording_segment(replay_id, project_id, filename, sequence_id):
+    with open(filename, "rb") as f:
+        file = File.objects.create(name=filename, type="application/octet-stream")
+        file.putfile(f)
+
+    ReplayRecordingSegment.objects.create(
+        replay_id=replay_id.replace("-", ""),
+        project_id=project_id,
+        sequence_id=sequence_id,
+        file_id=file.id,
+    )
+
+
+def make_filename(filename: str) -> str:
+    parent_dir = pathlib.Path(__file__).parent.resolve()
+    return f"{parent_dir}/rrweb-output/{filename}"
+
+
+def main():
+    project_name = "Replay Test"
+
+    if settings.SENTRY_SINGLE_ORGANIZATION:
+        org = Organization.get_default()
+        print(f"Mocking org {org.name}")  # NOQA
+    else:
+        print("Mocking org {}".format("Default"))  # NOQA
+        org, _ = Organization.objects.get_or_create(slug="default")
+
+    print(f"  > Mocking project {project_name}")  # NOQA
+    project, _ = Project.objects.get_or_create(
+        name=project_name,
+        defaults={
+            "organization": org,
+            "flags": Project.flags.has_releases,
+        },
+    )
+
+    replay_id = "44c586f7-bd12-4c1b-b609-189344a19e92"
+    seq1_timestamp = datetime.datetime.now() - datetime.timedelta(seconds=22)
+    seq2_timestamp = datetime.datetime.now() - datetime.timedelta(seconds=5)
+
+    print("Creating Clickhouse entries...")  # NOQA
+    store_replay(mock_replay(seq1_timestamp, project.id, replay_id, sequence_id=0))
+    store_replay(mock_replay(seq2_timestamp, project.id, replay_id, sequence_id=1))
+
+    print("Creating Postgres entries...")  # NOQA
```

Comment:
You could use click.echo() if you wanted to avoid the noqa.

Reply:
Thanks I updated it.

Also I just pushed minor changes to the test suite.  URL ingestion was merged upstream so I needed to assert that they existed in clickhouse.  Shouldn't affect any reviewing you might have done.